azext_iot/central/_help.py

```diff
@@ -1752,3 +1754,230 @@ def _load_central_edge_help():
             --device-id {deviceId}
             --children-ids {child_1} {child_2}
     """
+
+
+def _load_central_enrollment_group_help():
+    helps[
+        "iot central enrollment-group"
+    ] = """
+          type: group
+          short-summary: Manage and configure IoT Central enrollment group
+      """
+
+    helps[
+        "iot central enrollment-group list"
+    ] = """
+    type: command
+    short-summary: Get the list of enrollment groups in an application
+    examples:
+      - name: Get the list of enrollment groups in an application
+        text: >
+          az iot central enrollment-group list
+          --app-id {appid}
+    """
+
+    helps[
+        "iot central enrollment-group show"
+    ] = """
+    type: command
+    short-summary: Get details about an enrollment group by ID
+    examples:
+      - name: Get details about an enrollment group by ID
+        text: >
+          az iot central enrollment-group show
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group delete"
+    ] = """
+    type: command
+    short-summary: Delete an enrollment group by ID
+    examples:
+      - name: Delete an enrollment group by ID
+        text: >
+          az iot central enrollment-group delete
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group create"
```

Comment:
include help with how to make a sym key vs cert enrollment group

Reply:
Sure.